invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/ComputerTabletUploadsItem.js

```diff
@@ -0,0 +1,143 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import _truncate from "lodash/truncate";
+import React from "react";
+import { Button, Icon, Item, Label } from "semantic-ui-react";
+import { SearchItemCreators } from "../../utils";
+
+export const ComputerTabletUploadsItem = ({
+  result,
+  index,
+  editRecord,
+  statuses,
+}) => {
+  const access_status_id = _get(
+    result,
+    "ui.access_status.id",
+    i18next.t("open")
+  );
+  const access_status = _get(
+    result,
+    "ui.access_status.title_l10n",
+    i18next.t("Open")
+  );
+  const access_status_icon = _get(
+    result,
+    "ui.access_status.icon",
+    i18next.t("unlock")
+  );
+  const description_stripped = _get(
+    result,
+    "ui.description_stripped",
+    i18next.t("No description")
+  );
+  const title = _get(result, "metadata.title", i18next.t("No title"));
+  const creators = _get(result, "ui.creators.creators", []).slice(0, 3);
+  const subjects = _get(result, "ui.subjects", []);
+  const createdDate = _get(
+    result,
+    "ui.created_date_l10n_long",
+    i18next.t("No creation date found.")
+  );
+  const publicationDate = _get(
+    result,
+    "ui.publication_date_l10n_long",
+    i18next.t("No publication date found.")
+  );
+  const resource_type = _get(
+    result,
+    "ui.resource_type.title_l10n",
+    i18next.t("No resource type")
+  );
+  const version = result.ui?.version;
```

Comment:
`version` might be `undefined` and is later printed to the user as a `label`. 

Maybe set `version` to `''`?
```suggestion
  const version = result.ui?.version ?? '';
```